.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: '🐛 Bug report'
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in `remix-cookbook.com` (https://github.com/remix-cookbook/remix-cookbook.com).
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of `remix-cookbook.com` Discussions tab: https://github.com/remix-cookbook/remix-cookbook.com/discussions
+
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - `remix-cookbook.com` Issues tab: https://github.com/remix-cookbook/remix-cookbook.com/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
+         - `remix-cookbook.com` closed issues tab: https://github.com/remix-cookbook/remix-cookbook.com/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed
```

Comment:
NIT: `Closed` instead of `closed`

Reply:
Updated - https://github.com/remix-cookbook/remix-cookbook.com/pull/26/commits/b47d25038fdf41c72856178ad90dc5fa1d1a84b8